app/components/sdg/related_list_selector_component.html.erb

```diff
@@ -12,9 +12,16 @@
     <%= f.text_field :related_sdg_list,
                       class: "input",
                       placeholder: t("sdg.related_list_selector.placeholder"),
-                      label: t("sdg.related_list_selector.hint"),
+                      label:  t("sdg.related_list_selector.label"),
+                      hint: sanitize(t("sdg.related_list_selector.hint",
+                                      link: link_to(t("sdg.related_list_selector.help.text"),
+                                                      sdg_help_path,
+                                                      title: t("shared.target_blank"),
+                                                      target: "_blank")),
+                                     attributes: %w[href title target]),
                       data: { "suggestions-list": sdg_related_suggestions,
-                              "remove-tag-text": t("sdg.related_list_selector.remove_tag") } %>
+                              "remove-tag-text": t("sdg.related_list_selector.remove_tag") },
+                      "aria-label": t("sdg.related_list_selector.label") %>
```

Comment:
`aria-label` is redundant since we provide a real `label` tag :wink:. Use either one or the other.